etl/steps/data/garden/tuberculosis/2023-11-27/burden_estimates.meta.yml

```diff
@@ -0,0 +1,248 @@
+        unit: "%"
+        display:
+          numDecimalPlaces: 1
+      e_tbhiv_prct_lo:
+        title: Estimated HIV in incident tuberculosis (low bound)
+        unit: "%"
+        display:
+          numDecimalPlaces: 1
+      e_tbhiv_prct_hi:
+        title: Estimated HIV in incident tuberculosis (high bound)
+        unit: "%"
+        display:
+          numDecimalPlaces: 1
+      e_inc_tbhiv_100k:
+        title: Estimated incidence of tuberculosis cases who are HIV-positive per 100 000 population
+        unit: "per 100,000 individuals who are HIV-positive"
+        display:
+          numDecimalPlaces: 1
+      e_inc_tbhiv_100k_lo:
+        title: Estimated incidence of tuberculosis cases who are HIV-positive per 100,000 population (low bound)
+        unit: "per 100,000 individuals who are HIV-positive"
+        display:
+          numDecimalPlaces: 1
+      e_inc_tbhiv_100k_hi:
+        title: Estimated incidence of tuberculosis cases who are HIV-positive per 100,000 population (high bound)
+        unit: "per 100,000 individuals who are HIV-positive"
+        display:
+          numDecimalPlaces: 1
+      e_inc_tbhiv_num:
+        title: Estimated number of incident tuberculosis cases who are HIV-positive
+        unit: ""
+        display:
+          numDecimalPlaces: 0
+      e_inc_tbhiv_num_lo:
+        title: Estimated number of incident tuberculosis cases who are HIV-positive (low bound)
+        unit: ""
+        display:
+          numDecimalPlaces: 0
+      e_inc_tbhiv_num_hi:
+        title: Estimated number of incident tuberculosis cases who are HIV-positive (high bound)
+        unit: ""
+        display:
+          numDecimalPlaces: 0
+      e_mort_exc_tbhiv_100k:
+        title: Estimated mortality from all forms of tuberculosis excluding HIV-positive cases per 100,000 population
+        unit: ""
+        display:
+          numDecimalPlaces: 1
+      e_mort_exc_tbhiv_100k_lo:
+        title: Estimated mortality from all forms of tuberculosis excluding HIV-positive cases per 100,000 population (low bound)
+        unit: ""
+        display:
+          numDecimalPlaces: 1
+      e_mort_exc_tbhiv_100k_hi:
+        title: Estimated mortality from all forms of tuberculosis excluding HIV-positive cases per 100,000 population (high bound)
+        unit: ""
+        display:
+          numDecimalPlaces: 1
+      e_mort_exc_tbhiv_num:
+        title: Estimated number of deaths from all forms of tuberculosis excluding HIV-positive cases
+        unit: ""
+        display:
+          numDecimalPlaces: 0
+      e_mort_exc_tbhiv_num_lo:
+        title: Estimated number of deaths from all forms of tuberculosis excluding HIV-positive cases (low bound)
+        unit: ""
+        display:
+          numDecimalPlaces: 0
+      e_mort_exc_tbhiv_num_hi:
+        title: Estimated number of deaths from all forms of tuberculosis excluding HIV-positive cases (high bound)
+        unit: ""
```

Comment:
I am not sure if you actually wanted to add these units here so will not comment anymore but they are generally missing in other variables above too. Feel free to ignore me :)